js/webpack_get_defaults.js

```diff
@@ -17,10 +17,19 @@ const enums = require('./config_choices')
 const plugins = []
 
 /* Pre-configure loaders */
+
+var babelPresets = ['es2015', 'stage-3']
+var jsFileExtensionsRegexp = /\.js/
+
+if (config.build.profile === 'react') {
+  jsFileExtensionsRegexp = /\.jsx?$/
+  babelPresets.unshift('react')
+}
```

Comment:
there's already code in there that does `if (config.build.profile === 'angular') {`.

Maybe it's a good time to at least group these together in the file like:
```javascript
switch (config.build.profile) {
  case "angular":
    foo
    break
  [...]
}
```